frameworks/digital-outcomes-and-specialists/manifests/output_brief_response.yml

```diff
@@ -7,3 +7,13 @@
     - legacyDayRate
     - niceToHaveRequirements
     - legacyRespondToEmailAddress
+-
+  name: View ODS
```

Comment:
I think we want to have the View ODS section as a separate manifest named 'output_brief_response'.

The View response to requirements section should be moved into a manifest called 'legacy_output_brief_response'.